environment.yml

```diff
@@ -8,15 +8,18 @@ dependencies:
   - ipython
   - ocp
   - pyparsing
-  - sphinx=2.4
+  - sphinx=3.2.1
```

Comment:
We'll want to make sure that this version plays nice with the the 3D views in the docs.

Reply:
The docs build now in CI and it worked fine.